ws-nextjs-app/pages/[service]/send/[id]/FormContext/index.tsx

```diff
@@ -55,14 +59,26 @@ export const FormContextProvider = ({
 
   const handleSubmit = async (event: FormEvent) => {
     event.preventDefault();
+    setSubmitted(true);
 
     // Reset error state
     setSubmissionError(null);
 
     const formData = new FormData();
 
-    // TODO: This is a mock data, we should use the formState instead
-    Object.entries({ surname: 'BBC TEST NAME' }).forEach(([key, value]) => {
+    Object.entries(formState).forEach(([key, value]) => {
+      if (value === null) return;
+      if (value instanceof FileList) {
+        const fileList = value;
+        const fileListLength = fileList.length;
+
+        for (let fileIndex = 0; fileIndex < fileListLength; fileIndex += 1) {
+          const file = fileList.item(fileIndex);
+          if (file) formData.append(key, file);
```

Comment:
The checkbox component uses Boolean values to set the "checked" value on the input. 

May need stringified and parsed out at the component though if the form data doesn't accept it.

Reply:
Although looking at the docs, "checked" isn't really used to determine the value https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/checkbox#value

I mean, it can be technically but maybe it's not the official way of doing it 🤔